clients/callback_client.go

```diff
@@ -181,6 +198,15 @@ func (ts TranscodeStatus) String() string {
 
 // The various status messages we can send
 
+type RecordingEvent struct {
+	When        string `json:"when"`
```

Comment:
Likely just a personal preference, but WDYT of `event` or `type` for this field instead?

Reply:
good name